src/components/dashboard/FaceVerification/sdk/UICustomization.web.js

```diff
@@ -32,7 +32,21 @@ export const UITextStrings = {
   zoomResultIdscanUploadMessage: `Verifying you're\none of a kind`,
 
   toJSON() {
-    return mapKeys(pickBy(this, isString), (_, i18nString) => snakeCase(i18nString))
+    return keys(this).reduce((json, i18nString) => {
```

Comment:
@serdiukov-o-nordwhale I don't understand why we need this? BTW it is not working. The json var is not contained the required keys. I tried to quickly test what is wrong -  but didn't find where is a mistake.

Reply:
added a fix, please test again. if still fails - just revert to your working code (e.g. 'zoomRetryInstructionMessage1')

Reply:
the idea is the following. zoomRetryInstructionMessage1 looked a bit weird, so i tried to introduce the following format:
- if i18n key is a string - using it as is, modifying just a key (zoomResultIdscanUploadMessage -> zoom_result_idscan_upload_message
- if it's an array - we modifying a key and adding each item as <base key + index> (e.g. zoomRetryInstructionMessage -> zoom_retry_instruction_message_1, 2, 3, ...)

Reply:
It still fails. So I reverted changes to the working one.
```
omitBy(this, isFunction)
```
fails to load of Zoom api

Reply:
ok